cache/statement.h

```diff
@@ -0,0 +1,46 @@
+#ifndef CACHE_STATEMENT_H_
+#define CACHE_STATEMENT_H_
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "third_party/hiredis/hiredis.h"
+
+/**
+ * DEL command, mainly for testing
+ *
```

Comment:
I don't think it is appropriate to expose Redis-specific functions in `cache/statement`.

Reply:
The header should provide encapsulation of certain cache-oriented operations such as `get`, `set`, `remove`, `flush`, or `invalidate`.

Reply:
By the way, you might consider to encapsulate cache APIs with function pointer table technique as [reactor.h](https://github.com/embedded2016/server-framework/blob/master/reactor.h) and [reactor.c](https://github.com/embedded2016/server-framework/blob/master/reactor.c) did.